core/CronArchive/ReArchiveList.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Matomo - free/libre analytics platform
+ *
+ * @link https://matomo.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ *
+ */
+
+namespace Piwik\CronArchive;
+
+use Piwik\Concurrency\DistributedList;
+use Psr\Log\LoggerInterface;
+
+class ReArchiveList extends DistributedList
```

Comment:
it's great this was so easy to do and that we have this distributedlist already!